### DISCUSSION TITLE
Queue Gitlab Integration in the pipeline

### DISCUSSION THREAD

**Discussion by wolszczak96** [vercel/community#97] [Help] [answered]
Hi there.

Is it possible to configure Gitlab integration to wait with running it until the gitlab pipeline is finished? I would like to be able run **tests and stuff before deploying** to Vercel. I couldn't find the answer anywhere. Maybe it would be nice to add it to docs.

![Screenshot 2020-06-14 at 15 04 18](https://user-images.githubusercontent.com/25955276/84594130-5e3e9d80-ae50-11ea-9bb6-2af3cc6e4f6e.png)

**Comment by wolszczak96** [comment]
can I ping on this? 😳

**Comment by patoncrispy** [comment]
I'm here for this exact same issue. I looked into doing the deployments manually, but the vercel cli seems to require a login and I can't find documentation on how to do that during CI.

**Comment by darklight9811** [comment]
The topic has died but not my need for it

**Comment by Pieparker** [comment, marked as answer]
You'll find the best advice for this, along with any developments, in the [feature request](https://github.com/vercel/community/discussions/76). Although it was originally created with GitHub in mind, we'll capture all requests for queued Git-backed deploys there. Thanks!
